onnxruntime/test/providers/cpu/tensor/gather_elements_op_test.cc

```diff
@@ -269,9 +271,12 @@ void RunTypedTest<std::string>() {
   test8.Run();
 }
 
+// Disable TensorRT due to missing int8 calibrator
+#if !defined(USE_TENSORRT)
```

Comment:
we should not use USE_TENSORRT macros anymore.

Reply:
removed USE_TENSORRT macro